### ISSUE TITLE
Passing callback functions

### ISSUE THREAD

**Issue by AhmedSafi97** [GSG-G9/AO-auto-complete#16]
https://github.com/GSG-G9/AO-auto-complete/blob/be0e55b372e15add06c34e2e8168b87a183dd8f3/src/server.js#L7-L13

You can pass the function `router` directly as a callback to the `createServer` method.
```js
 http.createServer(router).listen(port, () => { 
   console.log(`Server is running on: http://${host}:${port}`); 
 });
```

**Comment by Osama-you** [comment]
What if we want to do a certain thing on the req before passing it to the router like checking if there is SQL injection or something like that

**Comment by AhmedSafi97** [comment]
You're right, but this is not your case so why would you worry about it! Actually, this is a topic for the next week: `middlewares`

**Comment by Osama-you** [comment]
ok
